src/main/java/org/folio/consortia/service/impl/UserAffiliationServiceImpl.java

```diff
@@ -149,6 +156,24 @@ private UUID getUserId(UserEvent userEvent) {
     return UUID.fromString(userEvent.getUserDto().getId());
   }
 
+  private void deletePrimaryAffiliationAndShadowUsers(UserEvent userEvent, String centralTenantId) throws JsonProcessingException {
+    log.info("Going to delete primary affiliation and all shadow users for userId: {}, tenant: {}",
+      userEvent.getUserDto().getId(), userEvent.getTenantId());
+    boolean existed = userTenantService.deletePrimaryUserTenantAffiliation(getUserId(userEvent));
+    if (!existed) {
```

Comment:
existed is little bit confusing. its name can be change to isPrimaryAffiliationExists
log can be improved by adding method names  deletePrimaryAffiliationAndShadowUsers:: Going to delete